ansible/roles/operator-pipeline/templates/openshift/tasks/set-env.yml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: set-env
```

Comment:
This is nice! It would be also great to extend it to support internal instances of Pyxis and re-use it hosted a release pipeline as well. If you have some spare time please do it, otherwise, file a story in a backlog. Thanks

Reply:
Updated. Please take one more look.